cob_bringup/robots/cob4-2.xml

```diff
@@ -204,10 +210,11 @@
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="pc" value="$(arg cob4-2-h1)" />
 		</include>
-		
-		<!--node pkg="cob_mimic" type="mimic_node.py" name="cob_mimic"/-->
+
 		<param name="/sound_controller/mode" value="cepstral" />
 		<include file="$(find cob_bringup)/drivers/sound.launch" />
+		
+		<include file="$(find cob_bringup)/drivers/mimic.launch" />
```

Comment:
can mimic now be launched from the current user?
is logging into the head display not required anymore?